client/src/components/AreasOfFocus/styles.css

```diff
@@ -11,96 +11,50 @@
 .area-text-section-red {
     border: 1.5px solid red;
     margin-bottom: 10px;
-    min-height: 100px;
+    min-height: 58px;
     display: flex;
     flex-wrap: wrap;
     padding: 5px;
+    overflow-y: auto;
 }
 
 .area-text-section-yellow {
-    border: 1.5px solid rgb(245, 190, 8);
+    border: 1.5px solid orange;
```

Comment:
Hi Ola, could you use the same colors of the bootstrapped buttons for the borders (All three of them) please?
Instead of: Red, Orange and Green.